supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,146 @@
+[[red-hat-single-sign-on-conventions]]
+
+[discrete]
+[[access-token]]
+== Access token
+A token that can be provided as part of an HTTP request that grants access to the service on which the request is  being invoked. This is part of the OpenID Connect and OAuth 2.0 specification.
+
+[discrete]
+[[assertion]]
+== Assertion
+Information about a user. This usually pertains to an XML blob that is included in a SAML authentication response that provided identity metadata about an authenticated user.
```

Comment:
I think the convention is to use full sentences.

Reply:
Thanks, @leswilliams44 
@ndeevy Can you make it  "An assertion provides information about a user."